logster/outputs/dogstatsd.py

```diff
@@ -0,0 +1,55 @@
+from logster.logster_helper import LogsterOutput
+import re
+import socket
+
+
+class DogstatsdOutput(LogsterOutput):
+    shortname = 'dogstatsd'
+
+
+    @classmethod
+    def add_options(cls, parser):
+        parser.add_option('--dogstatsd-host', action='store',
+                          help='Hostname and port for dogstatsd collector, e.g. statsd.example.com:8125')
+
+
+    def __init__(self, parser, options, logger):
+        super(DogstatsdOutput, self).__init__(parser, options, logger)
+        if not options.dogstatsd_host:
```

Comment:
i think you can achieve this with the `required` kwarg  to `add_option`

Reply:
From python on `optparse`

"required option
an option that must be supplied on the command-line; note that the phrase “required option” is self-contradictory in English. optparse doesn’t prevent you from implementing required options, but doesn’t give you much help at it either."